js/src/forum/index.js

```diff
@@ -1,17 +1,12 @@
 import { extend } from 'flarum/extend';
 import TextEditor from 'flarum/components/TextEditor';
-import icon from 'flarum/helpers/icon';
+import MarkdownArea from 'mdarea';
 
-let MarkdownArea;
-
-if (window.Reflect) {
-  require('@webcomponents/custom-elements');
-  require('@github/markdown-toolbar-element');
-  MarkdownArea = require('mdarea/mdarea.js');
-}
+import './pollyfills';
```

Comment:
Me thinks this should be `polyfills`. :smile:

Reply:
🙈